sdk/logical/events.go

```diff
@@ -12,6 +12,10 @@ import (
 
 // common event metadata keys
 const (
+	// EventMetadataPath is used in event metadata to show the API path the client must have the `subscribe` capability
+	// on in order consume the event. It is recommended that the event path metadata field uses that api path that was invoked
+	// in order to generate the event
+	EventMetadataPath = "path"
```

Comment:
Was this and the bus.go file mistakenly added? We should remove these as they are not relevant to the LDAP changes.

Reply:
yes it was mistakenly added! I thought i removed it right when I did it, will try to remove it again